### PR TITLE
initial push for falco add-on

### PR DESCRIPTION
**Pull request by parkand1**
Initial push for Falco add-on

### CODE REVIEW

File: lib/addons/falco/index.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ */
+ const defaultProps: HelmAddOnProps & FalcoAddOnProps = {
+    name: "blueprints-falco-addon",
+    namespace: "falco",
+    chart: "falco",
+    version: "1.17.4",
+    release: "falco",
+    repository:  "https://github.com/falcosecurity/charts",
+    createNamespace: true,
+    values: {
```
Comment (shapirov103) [inline lib/addons/falco/index.ts:45]: We generally don't set defaults for the values. You have properties like kubernetesSupportEnabled, etc. at the root level. Defaults should follow the same logic.

File: lib/addons/falco/index.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+/**
+ * populateValues populates the appropriate values used to customize the Helm chart
+ * @param helmOptions User provided values to customize the chart
+ */
+function populateValues(helmOptions: FalcoAddOnProps): Values {
+    const values = helmOptions.values ?? {};
+
+    setPath(values, "kubernetes.support.enabled",  helmOptions.kubernetesSupportEnabled);
```
Comment (shapirov103) [inline lib/addons/falco/index.ts:76]: The value may not be provided by the customer as `kubernetesSupportEnabled` is optional. You will overwrite your own default in the values. A better approach is to check for null and applying a default if not defined:

```
setPath(values, "kubernetes.support.enabled", helmOptions.kubernetesSupportEnabled ?? true); // or whatever the default is.
```

File: package.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
         "typescript": "~4.5.4"
     },
     "dependencies": {
+        "@aws-quickstart/eks-blueprints": "^0.1.4",
```
Comment (shapirov103) [inline package.json:30]: this is already in eks-blueprints, why do we need this dependency?

**Review by shapirov103 — COMMENTED** [review]
Looks great, a few comments.

File: lib/addons/falco/index.ts (hunk 1 above)
Comment (parkand1) [reply lib/addons/falco/index.ts:45]: addressed

File: lib/addons/falco/index.ts (hunk 2 above)
Comment (parkand1) [reply lib/addons/falco/index.ts:76]: addressed

File: package.json (hunk 3 above)
Comment (parkand1) [reply package.json:30]: removed - I ran into some weird dependency issues when i was testing but it ended up just being an issue with the version of CDK I was using

**Pull request merged**